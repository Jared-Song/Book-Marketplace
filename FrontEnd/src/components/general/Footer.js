import styles from "../../../styles/Home.module.css";
import Typography from "@material-ui/core/Typography";
import { Button, Link } from "@material-ui/core";
import Router from "next/router";

export default function Footer() {
  return (
    <>
      <footer className={styles.footer}>
        <br />
        <Typography>SEPT Team Project.</Typography>
        <Typography>
          Present by{" "}
          <Link
            className={styles.teamName}
            onClick={() => {
              Router.push("/teaminfo");
            }}
          >
            Moving House
          </Link>
          .
        </Typography>
      </footer>
    </>
  );
}
