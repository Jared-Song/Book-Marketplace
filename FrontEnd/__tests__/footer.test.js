import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";
import Footer from "../src/components/general/Footer";

it("renders Footer correctly", () => {
  const tree = renderer
    .create(
      <Footer />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

