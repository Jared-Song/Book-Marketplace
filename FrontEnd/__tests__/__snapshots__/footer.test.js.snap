// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer correctly 1`] = `
<footer>
  <br />
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    SEPT Team Project.
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    Present by
     
    <a
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
    >
      Moving House
    </a>
    .
  </p>
</footer>
`;
